### PR TITLE
Allow the extension for automatic future updates

### PR DESCRIPTION
**Pull request by sammyshj**
Changes done in the PR:
* Added Update.plist. This file allows for automatic future updates of the extension.

Note: This is also required for uploading the extension to the Safari Extensions Gallery.

### CODE REVIEW

File: privly.safariextension/Update.plist [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+			<key>CFBundleIdentifier</key>
+			<string>com.privly.privly</string>
+			<key>Developer Identifier</key>
+			<string>34B3M269R6</string>
+			<key>CFBundleVersion</key>
+			<string>0.1.0</string>
+			<key>CFBundleShortVersionString</key>
+			<string>0.1.0</string>
+			<key>URL</key>
+			<string>http://privly.github.io/assets/extensions/safari/privly.safariextz</string>
```
Comment (smcgregor) [inline privly.safariextension/Update.plist]: The domain should be https://priv.ly

The GitHub site is the staging site, meaning it serves the site but it is for testing/verification purposes.

Comment (sammyshj) [inline privly.safariextension/Update.plist]: Can you provide the URL for the Safari extension hosted on https://priv.ly?

Comment (smcgregor) [inline privly.safariextension/Update.plist]: It will be:

`https://priv.ly/assets/extensions/safari/privly.safariextz`

Comment (sammyshj) [inline privly.safariextension/Update.plist]: I have updated the code. Presently, the URL mentioned is not valid. We need to upload the extension to the URL specified.

**Comment by smcgregor** [comment]
So I will

1. merge this
2. re-build the extension
3. update the extension on privly.github.io
4. deploy priv.ly

sound good?

**Comment by sammyshj** [comment]
After merging this PR, there will be one more PR that I will open which ensures automatic updates for the extension. The next PR will depend on the URL of the current merged PR.
After both the PRs are merged, we can carry out steps 2, 3 and 4.
Thanks!

File: privly.safariextension/Update.plist [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Extension Updates</key>
+	<array>
+		<dict>
+			<key>CFBundleIdentifier</key>
+			<string>com.privly.privly</string>
+			<key>Developer Identifier</key>
+			<string>34B3M269R6</string>
```
Comment (smcgregor) [inline privly.safariextension/Update.plist]: I think this is your developer key? It should probably be `NW49B9S472` instead.

Comment (sammyshj) [inline privly.safariextension/Update.plist]: Updated both in `Update.plist` and `Info.plist`.

**Comment by sammyshj** [comment]
I have updated the code with the `Update Manifest URL` too now.

**Comment by smcgregor** [comment]
It should be live on priv.ly now.

**Comment by sammyshj** [comment]
Yes, it is. Thanks. We can now upload the extension to Safari Extensions Gallery.

**Pull request merged**